doc/reference/reference_lua/net_box.rst

```diff
@@ -820,9 +828,39 @@ With the ``net.box`` module, you can use the following
                                           be replaced by trigger-function
     :return: nil or function pointer
 
+..  _net_box-on_shutdown:
+
+..  function:: conn:on_shutdown([trigger-function[, old-trigger-function]])
+
+    Define a trigger for shutdown when a :ref:`box.shutdown <system-events_box-shutdown>` event is received.
+
+    The trigger starts in a new fiber.
+    While the ``on_shutdown()`` trigger is running, the connection stays active.
+    It means that it is allowed to send new requests from a trigger callback.
```

Comment:
The connection is allowed? The trigger? The user?

Reply:
Fixed